README.md

```diff
@@ -99,7 +99,7 @@ circumstances, you can use the `ProgressThresh` type:
 
 ```julia
 prog = ProgressThresh(1e-5, "Minimizing:")
-for val in logspace(2, -6, 20)
+for val in exp10.(range(2, stop=-6, length=20))
```

Comment:
Oh neat, @femtocleaner even makes adjustments to the README!  But perhaps a more straightforward example would work even better here...